cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1487,6 +1487,215 @@ action.summary_index._name = cyences
 action.summary_index.category = O365
 action.summary_index.alert_name = O365 - Application Change/Update
 
+[O365 - Login failure due to multi factor authentication]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	29 * * * *
+description = This alert will show the login failure due to multi factor authentication. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -85m@s
+dispatch.latest_time = -25m@s
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed (LogonError="DeviceAuthenticationRequired" OR LogonError="UserStrongAuthClientAuthNRequiredInterrupt") \
+| eval ExtendedProperties=mvzip('ExtendedProperties{}.Name','ExtendedProperties{}.Value'," : ") \
+| stats count, max(_time) as Last_Failed_Login, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user, ClientIP \
+| iplocation ClientIP \
+| fillnull Country, Region, City value="-" \
+| eval Location=ClientIP." (".count.") | ".Country." | ".Region." | ".City \
+| stats sum(count) as count, max(Last_Failed_Login) as Last_Failed_Login, list(Location) as Location, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user \
+| sort - count \
+| `cs_human_readable_time_format(Last_Failed_Login)` \
+| `cs_o365_failed_login_due_to_mfs_filter`
+action.summary_index = 1
+action.summary_index._name = cyences
+action.summary_index.category = O365
+action.summary_index.alert_name = O365 - Login failure due to multi factor authentication
+
+[O365 - Login failure outside US due to multi factor authentication]
```

Comment:
Replace the US with Home Country everywhere.

Reply:
Follow the Camel case for Alert naming convention.

Reply:
Both Change Done